src/clj/rems/api/services/workflow.clj

```diff
@@ -34,14 +34,16 @@
   (let [workflow (unrich-workflow (workflow/get-workflow id))
         workflow-body (cond-> (:workflow workflow)
                         handlers (assoc :handlers handlers))]
+    (util/check-allowed-organization! (:organization workflow))
     (db/edit-workflow! {:id id
                         :title title
                         :workflow (json/generate-string workflow-body)}))
   (applications/reload-cache!)
   {:success true})
 
-(defn set-workflow-enabled! [command]
-  (db/set-workflow-enabled! (select-keys command [:id :enabled]))
+(defn set-workflow-enabled! [{:keys [id enabled]}]
+  (util/check-allowed-organization! (:organization (workflow/get-workflow id)))
```

Comment:
Use the previous like this one?

Reply:
Let's talk about this in my follow-up PRs that clean up the services